modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIAdminImpl.java

```diff
@@ -146,37 +183,312 @@ public SettingsDTO getSettings() throws ApiException {
         return settingsApi.settingsGet();
     }
 
+    /**
+     * This method is used to create an application throttling policy.
+     *
+     * @param applicationThrottlePolicyDTO Application throttling policy DTO to be added.
+     * @return API response returned by the API call.
+     * @throws ApiException if an error occurs while creating the application throttling policy.
+     */
+    public ApiResponse<ApplicationThrottlePolicyDTO> addApplicationThrottlingPolicy(
+            ApplicationThrottlePolicyDTO applicationThrottlePolicyDTO) throws ApiException {
+
+        return applicationPolicyCollectionApi
+                .throttlingPoliciesApplicationPostWithHttpInfo(applicationThrottlePolicyDTO,
+                        Constants.APPLICATION_JSON_CONTENT_TYPE);
+    }
+
+    /**
+     * This method is used to retrieve an application throttling policy.
+     *
+     * @return API response returned by the API call.
+     * @throws ApiException if an error occurs while retrieving the application throttling policy.
+     */
+    public ApiResponse<ApplicationThrottlePolicyDTO> getApplicationThrottlingPolicy(String policyId)
+            throws ApiException {
+
+        return applicationPolicyIndividualApi
+                .throttlingPoliciesApplicationPolicyIdGetWithHttpInfo(policyId, null, null);
+    }
+
+    /**
+     * This method is used to update an application throttling policy.
+     *
+     * @param policyId                     Policy Id of the application throttling policy to be updated.
+     * @param applicationThrottlePolicyDTO Application throttling policy DTO which contains the update content.
+     * @return API response returned by the API call.
+     * @throws ApiException if an error occurs while updating the application throttling policy.
+     */
+    public ApiResponse<ApplicationThrottlePolicyDTO> updateApplicationThrottlingPolicy(String policyId,
+                                                                                       ApplicationThrottlePolicyDTO applicationThrottlePolicyDTO)
```

Comment:
Seems this exceeds the character limit

Reply:
Fixed with https://github.com/wso2/product-apim/pull/8750/commits/6d5d3d36ba4acc833b947ad9f83e3b19e3ad93c9